react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.js

```diff
@@ -1,11 +1,12 @@
 import "./PrimaryBtn.css";
+import cx from "classnames";
 
-function PrimaryButton({ title, handleSubmit }) {
+function PrimaryBtn({ className, title, onClick }) {
```

Comment:
A few things here:

* I renamed `handleSubmit` to `onClick` (see [this comment](https://github.com/tyreljackson-L904/React_Dev/pull/1/files#r715310988)). Because of that, I needed to update the name here so that I could consume it.
* I noticed you were passing a [`className`](https://github.com/tyreljackson-L904/React_Dev/pull/1/files#diff-de2e24014e2a0af0d54a710cef04a6afac3d53dd49eeda78a9161121f8c8fe93L50) to this component, but it was not consumed in this component, so it was lost. I pulled in `classnames` as you can see above so that I can compose the classNames in a nice way.